examples/async_callback_with_queue.py

```diff
@@ -118,6 +113,19 @@ async def main(args: argparse.Namespace):
         help="UUID of a characteristic that supports notifications",
     )
 
+    parser.add_argument(
+        "-d",
+        "--debug",
+        action="store_true",
+        help="sets the logging level to debug",
+    )
+
     args = parser.parse_args()
 
+    log_level = "DEBUG" if args.debug else "INFO"
```

Comment:
How about using the constants from the logging module instead of strings?

Reply:
yes, maybe it is better